deps/slfritchie_eflame-a08518142126f5fc541a3a3c4a04c27f24448bae/src/eflame.app.src

```diff
@@ -0,0 +1,13 @@
+{application, eflame,
```

Comment:
Please match the `deps/` path with the actual name of the app. The commit # already gives us which repo it's from.